examples/jsm/controls/OrbitControls.js

```diff
@@ -292,17 +288,35 @@ class OrbitControls extends EventDispatcher {
 
 		}();
 
-		this.dispose = function () {
+		this.connect = function (domElement) {
+
+			if ( domElement === document ) console.error( 'THREE.OrbitControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.' );
+
+			scope.domElement = domElement;
+			scope.domElement.style.touchAction = 'none'; // disable touch scroll
+
+			scope.domElement.addEventListener( 'contextmenu', onContextMenu );
+
+			scope.domElement.addEventListener( 'pointerdown', onPointerDown );
+			scope.domElement.addEventListener( 'pointercancel', onPointerCancel );
+			scope.domElement.addEventListener( 'wheel', onMouseWheel, { passive: false } );
+
+		};
+
+		this.disconnect = function () {
 
-			scope.domElement.removeEventListener( 'contextmenu', onContextMenu );
+			if (scope.domElement !== null) {
```

Comment:
We could log a warning if a consumer tries to `disconnect` a control that never has been connected, but not sure if it's worth or not

Reply:
No need for a warning in this case. Let's not make the class "too smart" ^^.